codacy/.helmignore

```diff
@@ -28,14 +28,3 @@
 
 ## addon values files and dev requirements
 requirements-dev.yaml
```

Comment:
We can now clean this up. Notice I'm not excluding `values-microk8s.yaml`. I think, like `values-production.yaml` it makes sense to bundle it in in the chart.